.github/workflows/rnmobile-ios-runner.yml

```diff
@@ -27,9 +27,25 @@ jobs:
               with:
                   show-progress: ${{ runner.debug == '1' && 'true' || 'false' }}
 
+            - name: Switch Xcode version to ${{ matrix.xcode }}
+              run: sudo xcode-select --switch /Applications/Xcode_${{ matrix.xcode }}.app
+
+            - name: Launch simulator
```

Comment:
I've moved this to the beginning using the `&` so it starts booting up in the background and doesn't block the rest of the steps.